src/app/components/forms/community/details/details.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+	<app-form name="detailsForm">
+		<app-form-group name="name" :label="$gettext(`Name`)">
+			<app-form-control
+				type="text"
+				:rules="{
+					max: 100,
+				}"
+				:disabled="!!formModel.game"
+			/>
+			<app-form-control-errors />
+		</app-form-group>
+
+		<app-form-group name="path" :label="$gettext(`URL Path`)">
+			<app-form-control-prefixed-input
+				prefix="gamejolt.com/c/"
```

Comment:
Can we do "environment.baseurl" here?

Reply:
i thought about it, but we dont have a const for it without the url scheme. didn't want to add it just for this and then update all the other consts to be based on it. annoying.